--- /lab/src/variants/A/HealthFunction.cs	2026-08-18 16:41:35.542573007 +0000
+++ /lab/src/variants/B/HealthFunction.cs	2026-08-18 16:42:13.807724675 +0000
@@ -23,6 +23,15 @@
         return response;
     }
 
+    [Function("Test")]
+    public HttpResponseData Test(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "test")] HttpRequestData request)
+    {
+        var response = request.CreateResponse(HttpStatusCode.OK);
+        response.WriteString("test");
+        return response;
+    }
+
     private static bool IsInstrumented() =>
         Environment.GetEnvironmentVariables().Keys.Cast<object>()
             .Select(key => key.ToString() ?? string.Empty)
